app/controllers/api/v2/template_controller.rb

```diff
@@ -13,7 +13,8 @@ class TemplateController < ::Api::V2::BaseController
         param :filter, String, :required => false, :desc => N_("Export templates with names matching this regex (case-insensitive; snippets are not filtered).")
         param :negate, :bool, :required => false, :desc => N_("Negate the prefix (for purging).")
         param :dirname, String, :required => false, :desc => N_("Directory within Git repo containing the templates.")
-        param :http_proxy_policy, ForemanTemplates.http_proxy_policy_types.keys, :required => false, :desc => N_("HTTP proxy policy for template sync. If you choose 'selected', provide the `http_proxy_id` parameter.")
+        param :http_proxy_policy, ForemanTemplates.http_proxy_policy_types.keys, :required => false, :desc => N_("HTTP proxy policy for template sync. \
+          This setting is only effective when synchronizing templates through the HTTP or the HTTPS protocol. If you choose 'selected', provide the `http_proxy_id` parameter.")
```

Comment:
I suppose that if you really don't want users to use this setting in other situations, you can be more direct with the wording:
```suggestion
          Use only when synchronizing templates through the HTTP or the HTTPS protocol. If you choose 'selected', provide the `http_proxy_id` parameter.")
```

Reply:
Thanks, applied.